oracle/src/events/processCollectedSignatures/index.js

```diff
@@ -46,6 +47,34 @@ function processCollectedSignaturesBuilder(config) {
         logger.info(`Processing CollectedSignatures ${colSignature.transactionHash}`)
         const message = await homeBridge.methods.message(messageHash).call()
 
+        if (config.accessLists) {
+          const parsedMessage = parseMessage(message)
+          const recipient = parsedMessage.recipient.toLowerCase()
+          const originalTxHash = parsedMessage.txHash
+          const allowanceList = await readAccessListFile(config.accessLists.allowanceList)
+          const blockList = await readAccessListFile(config.accessLists.blockList)
```

Comment:
Does it make sense to read the file once when the worker is started? I understand that it is good to have the file re-read automatically but if it is not changed to often so we will get it opened again and again every for transaction. This is especially not efficient if there is no file at all.

Reply:
Initially, I thought that it is required to support file updates in real time. But if it is not needed, then we can save file contents in memory.

Reply:
I would prefer to cache the content. The idea is that we will change the file not often (ideally allowance list must not be used at all) so it is OK to restart the service instead of having extra read cycles during the the worker operations.

Reply:
Added in-memory caching